src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -83,17 +83,30 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
   }
 
   get customSectionForms(): Array<CustomFormGroup | CustomFormArray> {
-    const customSections = [this.summary.body.form, this.summary.dimensions.form, this.summary.tyres.form, this.summary.weights.form];
-
     const type = this.vehicleTechRecord?.techRecord.find(record => record.statusCode === StatusCodes.CURRENT)?.vehicleType;
 
-    if (type === VehicleTypes.PSV) {
-      customSections.push(this.summary.psvBrakes!.form);
-    } else if (type === VehicleTypes.TRL) {
-      customSections.push(this.summary.trlBrakes!.form);
+    switch (type) {
+      case VehicleTypes.PSV:
+        return [
+          this.summary.body.form,
+          this.summary.dimensions.form,
+          this.summary.tyres.form,
+          this.summary.weights.form,
+          this.summary.psvBrakes!.form
+        ];
+      case VehicleTypes.HGV:
+        return [this.summary.body.form, this.summary.dimensions.form, this.summary.tyres.form, this.summary.weights.form];
+      case VehicleTypes.TRL:
+        return [
+          this.summary.body.form,
+          this.summary.dimensions.form,
+          this.summary.tyres.form,
+          this.summary.weights.form,
+          this.summary.trlBrakes!.form
+        ];
+      default:
+        return [];
     }
-
-    return customSections;
   }
```

Comment:
What is the cause and/or purpose of this change?

It does not appear to be within the scope of or related to 6640 or 7112. It takes 8 lines of code and expands them into 21, producing the same outcome. The only dynamic piece I see is the brakes form. The forms for body, dimensions, tyres and weights are duplicated within all of these cases.

Reply:
To fix the bug/console error where the app was reaching for these fields on the new vehicle types. For lgv, car and motorbike we want the empty array and if it goes looking for this stuff it gets confused

Reply:
Ok, now I understand.

Then you could do:
```typescript
isSmallVehicle(): boolean {
  const type = this.vehicleTechRecord?.techRecord.find(record => record.statusCode === StatusCodes.CURRENT)?.vehicleType;

  return type === VehicleTypes.LGV ||  type === VehicleTypes.CAR ||  type === VehicleTypes.MOTORCYCLE;
}

get customSectionForms(): Array<CustomFormGroup | CustomFormArray> {
  const customSections = !this.isSmallVehicle
    ? [this.summary.body.form, this.summary.dimensions.form, this.summary.tyres.form, this.summary.weights.form]
    : [];

  // ...
}
```